src/data/polynomial/laurent.lean

```diff
@@ -356,6 +363,106 @@ begin
     simpa using hn }
 end
 
+section support
+
+lemma support_C_mul_T (a : R) (n : ℤ) : (C a * T n).support ⊆ {n} :=
+by simpa only [← single_eq_C_mul_T] using support_single_subset
+
+@[simp] lemma to_laurent_support (f : R[X]) : f.to_laurent.support = f.support.image coe :=
```

Comment:
```suggestion
@[simp] lemma to_laurent_support (f : R[X]) : f.to_laurent.support = f.support.map sorry :=
```
Is likely to be more useful. The sorry might already exist, if not you can just use `⟨coe, int of_nat.inj⟩` or similar

Reply:
I used your version.  The proof can probably use some of the machinery of `finset.map`, but I simply recycled the old one.